pkg/ee/default-application-catalog/applicationdefinitions/falco-app.yaml

```diff
@@ -41,6 +41,13 @@ spec:
           chartVersion: 4.1.1
           url: https://falcosecurity.github.io/charts
     version: 0.37.0
+  - template:
+      source:
+        helm:
+          chartName: flux2
+          chartVersion: 2.14.1
+          url: https://fluxcd-community.github.io/helm-charts
+    version: 2.4.0
```

Comment:
Wrong application, you mistakenly updated the Falco app with a version from Flux.